### DISCUSSION TITLE
Does the android build also build the android samples?

### DISCUSSION THREAD

**Discussion by dboogert** [google/filament#4020] [Q&A] [answered]
Hi,

Follow the android build instructions here https://github.com/google/filament/blob/main/BUILDING.md
 
```shell
./build.sh -p android release
```
There don't seem to be any executable samples built in a similar fashion to say the linux build. Is this expected and samples need to be built as a separate process?

Cheers
Don

**Comment by romainguy** [comment, marked as answer]
The samples are not built by `build.sh`. You can either build them by opening the `android/` directory as an Android Studio project, or you can just run `./gradlew assembleDebug` from the `android/` directory.
